packages/formation/js/sidenav.js

```diff
@@ -126,18 +127,22 @@ class SideBarMenu {
     this.lastTabbableElement.removeEventListener('keydown', this.captureFocus);
   }
 
-  closeMenu() {
+  addCloseMenuListener() {
```

Comment:
needed to separate the event listener from the actual close menu function. This allows me to call the closeMenu without having to add a new event listener.